### ISSUE TITLE
Operation: OVERHAUL

### ISSUE THREAD

**Issue by Turtle84375** [Turtle84375/turtle84375.tk#41]
The CSS for the website is a mess, and the login system is a security vulnerability just waiting to happen, so I'll be redoing everything.

This repository will be archived and used as a reference for the new website, which will include a secure auth system and mobile support, to name a few features.

**Comment by Turtle84375** [comment]
Migration is in progress --> https://github.com/Turtle84375/turtle84375.me

**Comment by Quantum-Codes** [comment]
BTW /about and /about gives 404.. maybe fix that in new website if u put

**Comment by Turtle84375** [comment]
> BTW /about and /about gives 404.. maybe fix that in new website if u put

Yeah, there isn't actually an about page :P I never got around to making one

**Comment by Quantum-Codes** [comment]
Oops I wrote /about 2 times lol. I meant /blog in one of them too

**Comment by Turtle84375** [comment]
> Oops I wrote /about 2 times lol. I meant /blog in one of them too

Yeah lol, they'll both be a thing in the new site though 👀

**Comment by Quantum-Codes** [comment]
👍
